Source/Fuse.Scripting.JavaScript/FuseJS/Builtins.uno

```diff
@@ -75,10 +75,10 @@ namespace Fuse.Reactive.FuseJS
 
 		internal bool UpdateModules(Fuse.Scripting.Context context)
 		{
-			var activity = false;
 			if(_timer != null)
-				activity = _timer.UpdateModule(context) || activity;
-			return activity;
+				return _timer.UpdateModule(context);
+
+			return false;
```

Comment:
This cleanup could have been done separately, as it doesn't change any behavior.

Reply:
I've separate this into #811

Reply:
We should rebase on master, to get rid of this.